### DISCUSSION TITLE
Can there be an option to go back to editing the message

### DISCUSSION THREAD

**Discussion by abhi1203** [nyaruka/floweditor#1154] [Ideas]
Here, once we click on Whatsapp or Attachments, there's no clear way to go back to editing the message. Clicking somewhere at the top of the blue box takes the user back. but it's not intuitive. 

![image](https://user-images.githubusercontent.com/16714604/104171718-17864580-5429-11eb-9699-4c681951e858.png)

**Comment by ericnewcomer** [comment]
This is not something we've explicitly seen our users struggle with, but we'll leave this open for now to track this.
